first-read-last-write-cache/Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2021"
 
 [dependencies]
 thiserror = { workspace = true }
+proptest = "1.1.0"
```

Comment:
This should go to `[dev-dependencies]` and let's use the one declared in the `workspace`.